README.md

```diff
@@ -1 +1,68 @@
-# Weather-App
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project: Weather-App
+
+> A Front-end web application that fetches the current weather data about a specified location/region/country using the http://api.openweathermap.org/ API.
+
+![screenshot](./src/images/Screenshot.png)
+
+## Live demo
+
+🔗 [visit the page]()
```

Comment:
## Improve the live link
- [ ] The live link to where the page is hosted live appears to be broken. Users are currently being redirected to this [`link`](https://github.com/RICKCOYL/Weather-App/blob/feature), that is currently producing a `404` error message. Kindly provide a real live link to the page as this is a mandatory requirement for this project as explained in this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/).